static/app/types/integrations.tsx

```diff
@@ -302,6 +302,7 @@ export type SentryAppWebhookRequest = {
   webhookUrl: string;
   errorUrl?: string;
   organization?: {
+    id: string;
```

Comment:
is this being used?

Reply:
or was it just missing before and was being returned

Reply:
it's only being used for the dashboard

Reply:
update i made it optional so that nothing breaks while we switch to the new endpoint